docs/features/backup_and_restore.md

```diff
@@ -3,12 +3,190 @@ id: backup_and_restore
 title: Cumulus Backup and Restore
 hide_title: true
 ---
-
+<!-- markdownlint-disable MD029 -->
 # Cumulus Backup and Restore
 
 ## Deployment Backup and Restore
 
-Most of your Cumulus deployment can be recovered by redeploying via Terraform. However, the Cumulus metadata including providers, collections, granules, rules, and executions that is stored in [DynamoDB](./data_in_dynamodb) can only be restored if backup was configured or enabled. If a deployment is lost, logs and Step Function executions in the AWS console will be irrecoverable.
+Most of your Cumulus deployment can be recovered by redeploying via Terraform.
+However, the Cumulus metadata including providers, collections, granules, rules,
+and executions that are stored in [DynamoDB](./data_in_dynamodb), and
+concurrently being written to the Core Postgres instance can only be
+restored if backup was properly configured or enabled. If a deployment is lost,
+logs and Step Function executions in the AWS console will be irrecoverable.
+
+## Postgres Database
+
+### Please note
+
+* Cumulus supports a "bring your own" Postgres instance approach, however
+  our reference implementation utilizes a serverless Aurora/RDS database - as
+  such this reference provides AWS RDS Aurora Serverless backup options.
+
+* As of Core version 2.1+ Cumulus is transitioning from use of a DynamoDB backed
+  primary database to a Postgres instance with a reference Aurora/RDS Postgres
+  compatible module.     In this mode, the primary data records are still being
+  written to DynamoDB (and replicated to Elastic Search), however replicated
+  writes are being written to the required Postgres instance.  As such, the
+  primary source of recovery should be considered to be a re-migration of data
+  from the primary database.
+
+### Backup and Restore
+
+#### Re-migration
+
+In Cumulus v3.0.0+ the Postgres database is not the primary data store - as
```

Comment:
Should we specify here that we mean (3.0.0+ but < 4.0.0)?

Reply:
Possibly.   I was going with the 'when we do release v4.0.0 we need a ticket to update the disaster recovery docs' approach, but if we feel including those future plans won't be confusing here, I could see it.

Reply:
How about 6812210a2

Reply:
Maybe we should change it to version 5?